language/bert/basic/model/layers/pooler.py

```diff
@@ -0,0 +1,28 @@
+import torch
+import torch.nn as nn
+from .linear import Linear
```

Comment:
You rename the colossal linear as Colo_Linear. Distinguish it from the torch one.